Content.Server/Chemistry/ReagentEffects/HealthChange.cs

```diff
@@ -34,6 +34,8 @@ public sealed class HealthChange : ReagentEffect
         public override void Effect(ReagentEffectArgs args)
         {
             var scale = ScaleByQuantity ? args.Quantity : FixedPoint2.New(1);
+            if (args.MetabolismEffects != null)
+                scale *= (args.Quantity / args.MetabolismEffects.MetabolismRate);
```

Comment:
this seems like kind of a weird way to scale it. effects shouldnt really know about their metabolism rate, just how much they're metabolizing, which this already has an option to scale by. realistically it should just default to scaling by quantity (which would fix this), but i suppose that requires a lot more changes in yml

Reply:
>effects shouldnt really know about their metabolism rate, 

the status quo right now with reagents is that effects are consciously written with their metabolism rate in mind. I agree that's not ideal but a PR fixing that would be extremely large compared to this one